ckan/lib/helpers.py

```diff
@@ -948,10 +948,8 @@ def get_facet_items_dict(facet, limit=None, exclude_active=False):
     exclude_active -- only return unselected facets.
 
     '''
-    if not c.search_facets or \
-            not c.search_facets.get(facet) or \
-            not c.search_facets.get(facet).get('items'):
-        return []
+    if not hasattr(c, 'search_facets'):
```

Comment:
This should be the other way around right? `if hasattr(c, 'search_facets'):`

Reply:
i thought it should return empty list if search_facets is missing ?